### ISSUE TITLE
[Bug] IsTranslucent iOS with ScrollView issue

### ISSUE THREAD

**Issue by skordesign** [xamarin/Xamarin.Forms#7096]
### Description
Set IsTranslucent = true and use ScrollView will not working.

### Steps to Reproduce

1. Custom renderer NavigationPage and set NavigationBar.IsTranslucent = true
2. On Forms view use ScrollView as Content of ContentPage
3. Start app

### Expected Behavior
The Content of ContentPage shouldn't has Padding on top

### Actual Behavior
The Content of ContentPage has Padding on top.

### Basic Information

- Version with issue: Xamarin.Forms 3.6.0 
- Last known good version: Unknown
- IDE: Visual Studio 2019
- Platform Target Frameworks: 
  - iOS:  12.2

### Screenshots
![image](https://user-images.githubusercontent.com/18341740/62673851-28c45a80-b9ca-11e9-8a02-78f84f53fcbe.png)

**Comment by jfversluis** [comment]
Hi @skordesign 

Could you please try this with the latest Xamarin.Forms version? Also, if you could put this into a small project that reproduces this behavior we would be very grateful!

Thank you!

**Comment by skordesign** [comment]
Hi @jfversluis
You can use it repo: https://github.com/skordesign/TranslucentNav
I've updated to Xamarin.Forms latest version but nothing change.

**Comment by jfversluis** [comment]
As far as I can tell you are using the example as written here: https://xamgirl.com/transparent-navigation-bar-in-xamarin-forms/

The one thing that seems to be missing from your code is in this file:

https://github.com/CrossGeeks/Forms-TransparentNavigationBar/blob/master/TransparentNavBarXForms/MainPage.xaml

As you can see the Image has a negative margin of 40 to pull up the image to accommodate for the empty space the NavigationBar leaves behind.

If this is not what you meant, please feel free to reopen the issue. Thanks!

**Comment by skordesign** [comment]
The problem is when I use Image as Content of ContentPage, it doesn't have any padding on top
Just set content like this:
```xml
<Image Source="https://puu.sh/BxiB5/f0c4b71b37.png"
           VerticalOptions="FillAndExpand"
           HorizontalOptions="FillAndExpand"
           Aspect="AspectFill" />
```

**Comment by jfversluis** [comment]
I'm not sure if I understand you correctly. What is happening here is that normally the navigation bar would show there. If you make it translucent you will also have to tell the rest of the content that it should behave differently. Making the navigation bar translucent doesn't hide it, it is still there and taking up space.

If you want some kind of padding, take a smaller margin or add some padding yourself. In any case, you have to come up with something to make the right layout, the behavior from what I can see is working as intended.

**Comment by skordesign** [comment]
Ok, let's talk about my problem again.
I have a ContentPage, and when I set its Content = Image, it's not has the top padding (as I expected).
But when I set Content = ScrollView (Image inside the ScrollView) the top padding showed up. And I don't want it.
What I want is how to set Content = ScrollView(Image inside) and top padding not show up.

The padding tops are different when with/without ScrollView, that's it. Is it normal?
